pages/glossary/programmatically-determined-context.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Programmatically determined context
+key: programmatically-determined-context
+---
+
+The programmatically determined context of a piece of information (_e.g._ an element) is all the additional information that has a [programmatically determinable](https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable) [relationship](https://www.w3.org/TR/WCAG21/#dfn-relationships) to it.
+
+It includes, but is not restricted to, `aria-label` attribute, element that is referred by `aria-labelledby` or `aria-describedby` attributes, the `title` attribute, ancestor `label` elements, `label` elements referring the current element with a `for` attribute, ancestor elements, table header associated with the cell containing the information, headings, text inside the same paragraph (`p` element), sentence, table cell, list item, …
```

Comment:
I don't think title and aria-label are part of an element's context. I can see an argument for aria-labelledby and for labels, but I disagree with calling attributes part of the context.

Reply:
I don't agree. I think the accessible name should be part of the programmatically determined context of an element.